MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs

```diff
@@ -1571,7 +1589,12 @@ public string GetGraphicalSubtitleParam(EncodingJobInfo state, EncodingOptions o
                     state.VideoStream.Width.Value,
                     state.VideoStream.Height.Value);
 
-                videoSizeParam += ":force_original_aspect_ratio=decrease";
+		if (string.Equals(outputVideoCodec, "h264_qsv", StringComparison.OrdinalIgnoreCase))
+		{
+		    videoSizeParam += ",hwupload=extra_hw_frames=64";
+		} else {
+	            videoSizeParam += ":force_original_aspect_ratio=decrease";
```

Comment:
this is secretly putting separate feature into single PR, please don't :)

Reply:
Can you please elaborate further? It's required for software decode -> hardware encode